Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -113,6 +113,22 @@ extension MongoClient {
 
         return true
     }
+
+    internal convenience init(options: ClientOptions? = nil) throws {
+        try self.init(connectionString: getConnStr(), options: options)
+    }
+
+    internal convenience init() throws {
+        try self.init(connectionString: getConnStr())
+    }
+}
+
+func getConnStr() -> String {
+    if let connStr = ProcessInfo.processInfo.environment["MDB_CONN_STR"] {
```

Comment:
libmongoc uses [`MONGOC_TEST_URI`](https://github.com/mongodb/mongo-c-driver/blob/master/CONTRIBUTING.md#testing), alongside many other similarly named options. PHP uses [`MONGODB_URI`](https://github.com/mongodb/mongo-php-driver/blob/master/CONTRIBUTING.md#local-mongo-orchestration-and-travis-ci).

I vote for renaming this to `MONGODB_URI`.

Reply:
Yeah, I think this is a good justification for changing the name. I don't really know much about what the other drivers are doing but having consistency across them is better than having a new variable name. I've changed this to agree with what PHP is using.